cherryontech/ywgc
# Fix services

## Refactoring the services component

https://ywgc.atlassian.net/browse/YWGC-38?atlOrigin=eyJpIjoiMGYwZGU3ZjRhOWFjNGY3NDk2ZmQzNDY2NzQ5ZGJjODAiLCJwIjoiaiJ9

## Work done
- Made all the changes based on the new design on figma (add horizontal line, make sure the font size are switching to the right font size when it comes to a different screen size)

## Manual testing instructions
- Responsive to screen size.
- Make sure the padding/margin are different when it's web screen and mobile screen.
- Make sure the icons are align in the center for mobile version in columns and align in center for web screen in rows.
- All of the fonts are set up to be in a different size when it comes to a different screen size like web screen and mobile screen
- Check whether the horizontal line (underline for Services title) change its size when it's mobile. 

## Gotchas/What I learned
To be more specific in the manual testing.
